library/rotations/src/Tests.qs

```diff
@@ -0,0 +1,50 @@
+import HammingWeightPhasing.HammingWeightPhasing;
+import Std.Diagnostics.CheckOperationsAreEqual;
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Looks like the imports got a strange ordering here in relations to the copyright comment. I know this can happen when the imports are added automatically with auto-complete.